lib/model/katex.dart

```diff
@@ -136,6 +154,110 @@ class _KatexParser {
   KatexNode _parseSpan(dom.Element element) {
     // TODO maybe check if the sequence of ancestors matter for spans.
 
+    final spanClasses = List<String>.unmodifiable(element.className.split(' '));
```

Comment:
nit: move this to where we might start to use it, since it looks like vlist handling doesn't require `spanClasses`